en/04_Changelogs/5.1.0.md

```diff
@@ -48,6 +49,16 @@ Note that even if you store the schema in the cache directory, the schema still
 
 See [choosing where the cache is stored](/developer_guides/graphql/getting_started/building_the_schema/#cache-location) for more information about configuring the schema storage directory.
 
+### CMS page search performance
+
+- The CMS search has been optimised to reduce the number of database queries made when searching for pages when the site uses elemental content blocks. This has resulted in a small performance improvement.
+- A new opt-in behaviour is available that makes a very large different to the performance when using elemental content blocks. In testing this more than doubled performance. The opt-in disables the default behaviour of rendering all content blocks for CMS search, and instead simply extracts the database contents of the elements. Note this does not use the `$searchable_fields` config. To opt-in to this behaviour, use the following config:
```

Comment:
If they are any downside to the behaviour, can we make them explicit?

Reply:
Have added `There is a downside to consider which is that any related content not directly on the element will not be matched against the search query.`